src/components/home/Free.js

```diff
@@ -15,21 +15,52 @@ function Free() {
   };
 
   return (
-    <Fragment>
+    <div id="free-card-container">
       <div className="text-container">
-        <h1>Free. &nbsp;For anyone and everyone.</h1>
+        <h1>Free. &nbsp;For everyone.</h1>
         <p>
-          Node Warrior is developed by a group of passionate software
-          engineers who believe that quality computer science resources
-          should be available to all.
+          Destructure.io is a platform created by software engineers who believe quality programming resources
+          should be accessible to anyone, anywhere.
         </p>
+        <div style={{ "display": "inline-block" }}>
```

Comment:
We should probably do this in CSS but eh its late

Reply:
I thought about that too, but that's literally the only style this div needs and I couldn't think of a good class name for it 😅

Reply:
idk you could literally just call it inline-block if you really want

Reply:
Lol actually yeah I should because I remembered I do this elsewhere too.